src/test/regress/expected/multi_mx_function_call_delegation.out

```diff
@@ -499,12 +499,28 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT test.x, r.cou
 
 -- Test that we don't propagate to non-metadata worker nodes
 select stop_metadata_sync_to_node('localhost', :worker_1_port);
+DEBUG:  relation 20567 does not exist, skipping
```

Comment:
These relations DO exist in `pg_dist_partition` on the worker. However, they are not included in `pg_class`. When we send the drop command to workers (for the partitions on the workers), our UDF `worker_drop_distributed_table` tries to drop these but errors out because `ResolveRelationId` returns `InvalidOid`, since their relationIds are not found.

I think, here we can avoid printing these log messages by changing the log level. We can create a separate issue for this.